### PR TITLE
Make building easier on Windows by setting PowerShell execution policy

### LINKED ISSUE #6617: [Bug] Make building easier on Windows by setting PowerShell execution policy
### Description

Executing arbitrary PowerShell files typically results in an error.

By adding a `CMD` file that calls the PowerShell file, it can be executed with an execution policy that allows it to be run. This is what other repos such as ASP.NET Core do: [`build.cmd`](https://github.com/aspnet/AspNetCore/blob/master/build.cmd) launches PowerShell with an execution policy that can run [`build.ps1`](https://github.com/aspnet/AspNetCore/blob/master/build.ps1).

### Steps to Reproduce

1. Clone repo on Windows machine
2. Try to run `./build.ps1`

### Expected Behavior

It should build.

### Actual Behavior

You get an error:

```
PS C:\GitHub\Xamarin.Forms> .\build.ps1
.\build.ps1 : File C:\GitHub\Xamarin.Forms\build.ps1 cannot be loaded. The file C:\GitHub\Xamarin.Forms\build.ps1 is
not digitally signed. You cannot run this script on the current system. For more information about running scripts and
setting execution policy, see about_Execution_Policies at https:/go.microsoft.com/fwlink/?LinkID=135170.
At line:1 char:1
+ .\build.ps1
+ ~~~~~~~~~~~
    + CategoryInfo          : SecurityError: (:) [], PSSecurityException
    + FullyQualifiedErrorId : UnauthorizedAccess
```

### Basic Information

N/A

### PR DESCRIPTION
**Pull request by Eilon**
### Description of Change ###

Add `build.cmd` that launches `build.ps1` with the proper execution policy so that it doesn't insta-fail.

### Issues Resolved ### 

Fixes #6617 

### API Changes ###

None

### Platforms Affected ### 

N/A

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###

Build on Windows.

### PR Checklist ###

- [ ] Has automated tests 
- [X] Rebased on top of the target branch at time of PR
- [X] Changes adhere to coding standard

### CODE REVIEW

**Comment by Eilon** [comment]
@rmarinho / @PureWeen - please let me know if you have any thoughts on this PR. I'm hoping this will make it even easier for Windows-based contributors to work on this repo.

File: build.cmd [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@ECHO OFF
+SETLOCAL
+PowerShell -NoProfile -NoLogo -ExecutionPolicy Bypass -Command "[System.Threading.Thread]::CurrentThread.CurrentCulture = ''; [System.Threading.Thread]::CurrentThread.CurrentUICulture = ''; try { & '%~dp0build.ps1' %*; exit $LASTEXITCODE } catch { write-host $_; exit 1 }"
```
Comment (PureWeen) [inline build.cmd]: My PS chops aren't the best :-) 

What does adding these help with?
```
 "[System.Threading.Thread]::CurrentThread.CurrentCulture = ''; [System.Threading.Thread]::CurrentThread.CurrentUICulture = '';
```

Most samples I looked up for doing this type of operation from CMD prompt didn't have these as part of it.

**Comment by PureWeen** [comment]
@Eilon can you add some commented out examples in build.cake file

https://github.com/xamarin/Xamarin.Forms/blob/master/build.cake#L9

of the syntax someone running this *.cmd* would use to include target properties?

**Comment by Eilon** [comment]
Great questions, I will look into this. I copied it from the ASP.NET Core repo, which is tried & true, but I should figure this out anyway.

**Comment by PureWeen** [comment]
> Great questions, I will look into this. I copied it from the ASP.NET Core repo, which is tried & true, but I should figure this out anyway.

link? I was looking around for some other examples of this and most of them matched yours pretty closely but didn't have the threading part so was just curious about it

ReactiveUI has a version of this as well
https://github.com/reactiveui/ReactiveUI/blob/master/build.cmd

But it seems like what you have here might have simpler execution syntax

**Comment by Eilon** [comment]
All the ASP.NET Core ones do this:

1. https://github.com/aspnet/AspNetCore/blob/master/build.cmd
2. https://github.com/aspnet/AspNetCore/blob/master/clean.cmd
3. https://github.com/aspnet/AspNetCore/blob/master/restore.cmd

**Comment by Eilon** [comment]
It could be that it's unnecessary. The commit that added those doesn't seem to say why: https://github.com/aspnet/AspNetCore/commit/510dab8407557ba07b443ae1062915d3a1f45443#diff-02389dfac0a172ab80211625efc34318

**Comment by Eilon** [comment]
I'm fine removing it; it doesn't *seem* necessary to set the thread culture. Presumably any given build task should set the thread culture themselves if they actually need it.

**Comment by PureWeen** [comment]
We can leave it since they have it :-) 
Can you just add some quick samples of how users would execute this command and include cake arguments?

https://github.com/xamarin/Xamarin.Forms/pull/6618#issuecomment-506878361

Then we can merge

**Comment by Eilon** [comment]
Will do.

**Comment by Eilon** [comment]
OK updated PR w/ CMD + PS1 samples. FYI: The syntax is exactly the same because all the CMD does is pass the parameters to the PS1 file verbatim. The `%*` token in CMD means "all command line parameters".

**Comment by PureWeen** [comment]
> FYI: The syntax is exactly the same because all the CMD does is pass the parameters to the PS1 file verbatim.

Awesome!! I was hoping this was the case

**Comment by PureWeen** [comment]
When you run this from cmd what version of Visual Studio does it use for you? 

By default when I use the *build.cmd* it uses VS 2017 for me but it fails with not being able to find the Android SDK directory. Though I have my directory in a non standard place.

If I force the cake file to 2019 then it all compiles fine. I'm just curious your experience

**Comment by Eilon** [comment]
I'm abandoning this PR; it's replaced by https://github.com/xamarin/Xamarin.Forms/pull/6755.

**Pull request closed without merging**